model/src/main/scala/explore/model/ProposalDetails.scala

```diff
@@ -5,20 +5,27 @@ package explore.model
 
 import cats._
 import explore.model.enum._
+import lucuma.core.model.Partner
 import lucuma.core.model.StandardUser
 import monocle.macros.Lenses
 
 @Lenses
 final case class ProposalDetails(
-  title:         String,
-  pi:            StandardUser,
-  proposalClass: ProposalClass,
-  category:      TacCategory,
-  toOActivation: ToOActivation,
-  keywords:      Set[Keyword],
-  abstrakt:      String
+  title:          String,
+  pi:             StandardUser,
+  proposalClass:  ProposalClass,
+  category:       TacCategory,
+  toOActivation:  ToOActivation,
+  keywords:       Set[Keyword],
+  abstrakt:       String,
+  partnerSplits:  List[PartnerSplit],
+  band1And2Hours: Double,
+  band3Hours:     Double
 )
```

Comment:
I was going to ask exactly that but also thought it could be overkill. I guess it is up to you

Reply:
I did this to see what it would be like, and I think it's OK. You just end up with some `field.value` and `field.value.value` some places in your code. I'll put in a commit with it after I finish up the current UX stuff I'm working on.

One question I did have is how using coulomb/refined will interact with graphQL. I'll start a conversation in slack.